dev-libs/opencl-clang/opencl-clang-8.0.0_p1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+CLANG_VERSION=8
+MY_PV="${PV//_p/-}"
+
+DESCRIPTION="Thin wrapper library around clang for OpenCL"
+HOMEPAGE="https://github.com/intel/opencl-clang"
+SRC_URI="https://github.com/intel/opencl-clang/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="UoI-NCSA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+# depend only on opencl-clang to pull in the right slots of clang and llvm
+DEPEND="=dev-util/spirv-llvm-translator-${CLANG_VERSION}*"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+src_prepare() {
+	default
+
+	cmake-utils_src_prepare
+
+	# we install clang in a separate top-dir, but this build system
+	# assumes clang below llvm
+	sed -i \
+		-e "s|\${LLVM_LIBRARY_DIRS}|/usr/$(get_libdir)|" \
```

Comment:
What's this path to? It looks really suspicious because clang doesn't install anything to /usr/lib64.

Reply:
as written, the build system assumed the whole clang installation tree to be below the llvm tree `/usr/lib/llvm/8/clang/8.0.0`, instead we have `/usr/lib/clang/8.0.0`.

Reply:
Yes, we do. We have `lib`, not `$(get_libdir)`. Also please upgrade your system to 17.1 profiles.

Reply:
> Yes, we do. We have `lib`, not `$(get_libdir)`.

Ok, will change.

> Also please upgrade your system to 17.1 profiles.

No can do. This is a production system (except for the OpenCL part).